eos-updater/tests/meson.build

```diff
@@ -2,6 +2,10 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 # python-dbusmock doesn’t currently provide a way to install out-of-tree templates
```

Comment:
This comment is outdated now.

Reply:
Fixed, not sure how I missed that :facepalm: